docs/methoddocs/utils.md

```diff
@@ -4,4 +4,5 @@
 .. automodule:: ape.utils
     :members:
     :show-inheritance:
+    :exclude-members: abstractmethod, dataclass
```

Comment:
These borrowed methods docs don't load correctly for us and trying to overwrite them causes other issues